### DISCUSSION TITLE
Made in Lapis

### DISCUSSION THREAD

**Discussion by leafo** [leafo/lapis#749] [Show and tell]
Share links to your projects made in Lapis, with source code if available.

Here are some things I've made:

  * <https://luarocks.org> - [source](https://github.com/leafo/moonrocks-site)
  * <https://itch.io>
  * <https://streak.club> - [source](https://github.com/leafo/streak.club)
  * <https://sightreading.training> - [source](https://github.com/leafo/sightreading.training)
  * [Ludum Dare game browser](http://ludumdare.itch.io) - [source](https://github.com/leafo/ludum-dare-browser)

**Comment by solisoft** [comment]
https://fasty.ovh  - [source](https://github.com/solisoft/fasty)

**Comment by cycomachead** [comment]
https://snap.berkeley.edu - [source](https://github.com/snap-cloud/snapCloud)
